src/lib/notificationWorker.js

```diff
@@ -15,6 +15,7 @@ class NotificationWorker {
     this.Transfer = Transfer
     this.Subscription = Subscription
     this.Fulfillment = Fulfillment
+    this.config = require('../services/config')
```

Comment:
We want to avoid requiring stateful modules (anything in src/services) in any of the src/lib modules. There are two other options I considered instead of having a `require` here.

1) Pass the service/config into the constructor
2) Require lib/config in this module, and load a config object for every instance of NotificationWorker.

I think option 2 is nicer, but I want to get hear any opinions on this.